src/index.js

```diff
@@ -7,7 +7,11 @@ const { server: { port, host } } = require('./config');
 require('dotenv').config();
 
 // Database connection
-require('./helpers/databaseConnection');
+require('./helpers/databaseConnection')
+  .then(
```

Comment:
This .then chain would look a lot nicer on the helpers export rather than on a require.

Reply:
I agree! I did not know you were able to do that. Thanks